spring-kafka/src/main/java/org/springframework/kafka/listener/adapter/RecordFilterStrategy.java

```diff
@@ -58,4 +60,16 @@ default List<ConsumerRecord<K, V>> filterBatch(List<ConsumerRecord<K, V>> record
 		return records;
 	}
 
+	/**
+	 * Determine whether {@link FilteringBatchMessageListenerAdapter} should invoke
+	 * the {@link BatchMessageListener} when all {@link ConsumerRecord}s in a batch have been filtered out
+	 * resulting in empty list. By default, do invoke the {@link BatchMessageListener} (return false).
+	 * @return true for {@link FilteringBatchMessageListenerAdapter} to {@link BatchMessageListener}
```

Comment:
```
{@link FilteringBatchMessageListenerAdapter} to {@link BatchMessageListener}
-> {@link FilteringBatchMessageListenerAdapter} to not invoke {@link BatchMessageListener}
```
`not invoke` is correct doc?

Reply:
Fixed: https://github.com/spring-projects/spring-kafka/commit/5c3412204517303aacdbabdbb1c2b62866d3d9f2.
Thank you!